test/unitary/service/pre-post/pre.test.ts

```diff
@@ -0,0 +1,52 @@
+import "reflect-metadata";
+import { constants } from "@bc/domain/constants";
+import { EntryPoint } from "@bc/domain/entry-point";
+import { BaseConfiguration } from "@bc/service/config/base-configuration";
+import { PreExecutor } from "@bc/service/pre-post/pre";
+import { Pre } from "@kie/build-chain-configuration-reader";
+import Container from "typedi";
+import { ExecuteCommandService } from "@bc/service/command/execute-command-service";
+import * as core from "@actions/core";
+
+// just for initialization otherwise not relevant to testing
+Container.set(constants.CONTAINER.ENTRY_POINT, EntryPoint.GITHUB_EVENT);
+
+// disable logs
+jest.spyOn(global.console, "log");
+jest.spyOn(core, "startGroup").mockImplementation(() => undefined);
+jest.spyOn(core, "endGroup").mockImplementation(() => undefined);
+
+test.each([
+  ["single command", "cmd", ["cmd"]],
+  ["multiple commands", ["cmd1", "cmd2"], ["cmd1", "cmd2"]],
+])("%p", async (_title: string, cmds: Pre, executedCmds: string[]) => {
+  jest.spyOn(BaseConfiguration.prototype, "definitionFile", "get").mockImplementation(() => {
```

Comment:
post.ts relies on `configurationservice.getPre` method to get the post commands, so that's the method to mock from here. Consequently, the `configurationservice.getPre` needs to be covered by a new test.